lms/static/sass/shared/_footer.scss

```diff
@@ -5,6 +5,7 @@
 
 .wrapper-footer {
   @extend %ui-print-excluded;
+  margin-top: ($baseline*2);
```

Comment:
NIT: This rule should ideally be handled by the Sass ``_layouts.scss`` partial in an ideal world, since it pertains to overall view layouts and there's already a ``.wrapper-footer`` definition. No action needed in the LMS styling jungle, just FYI.